2022q4/freebsd-foundation.adoc

```diff
@@ -4,54 +4,38 @@ Links: +
-material to promote, educate, and advocate for the FreeBSD Project, facilitate
-collaboration between commercial vendors and FreeBSD developers, and finally,
-represent the FreeBSD Project in executing contracts, license agreements, and
-other legal arrangements that require a recognized legal entity.
+The FreeBSD Foundation is a 501(c)(3) non-profit organization dedicated to supporting and promoting the FreeBSD Project and community worldwide.
+Donations from individuals and corporations are used to fund and manage software development projects, conferences, and developer summits.
+We also provide travel grants to FreeBSD contributors, purchase and support hardware to improve and maintain FreeBSD infrastructure, and provide resources to improve security, quality assurance, and release engineering efforts.
+We publish marketing material to promote, educate, and advocate for the FreeBSD Project, facilitate collaboration between commercial vendors and FreeBSD developers, and finally, represent the FreeBSD Project in executing contracts, license agreements, and other legal arrangements that require a recognized legal entity.
 
 ==== Fundraising Efforts
 
-Thank you to everyone who made a financial contribution in 2022! We're still
-tallying up the totals and will have final numbers soon. Unfortunately, we did
-not meet our fundraising goal, which reinforced our need of having someone who
-can focus on encouraging organizations to invest in FreeBSD. We will bring
-someone on board soon to help with that effort.
-
-In this Quarterly Status report you'll read about many of the areas we funded in
-Q4 to improve FreeBSD and advocate for the Project (the two main areas we spend
-money on). Check out reports on the internally and externally funded projects
-like Openstack on FreeBSD, Enabling Snapshots on Filesystems Using Journaled
-Soft Updates, FreeBSD as a Tier 1 cloud-init Platform, RISC-V support, and Arm64
-support. In addition, we provided tons of community engagement and education
-opportunities virtually and in-person!
-
-If you want to help us continue our efforts, please consider making a donation
-towards our 2023 fundraising campaign!
+Thank you to everyone who made a financial contribution in 2022! We're still tallying up the totals and will have final numbers soon.
+Unfortunately, we did not meet our fundraising goal, which reinforced our need of having someone who can focus on encouraging organizations to invest in FreeBSD.
+We will bring someone on board soon to help with that effort.
+
+In this Quarterly Status report you'll read about many of the areas we funded in Q4 to improve FreeBSD and advocate for the Project (the two main areas we spend money on).
+Check out reports on the internally and externally funded projects like Openstack on FreeBSD, Enabling Snapshots on Filesystems Using Journaled Soft Updates, FreeBSD as a Tier 1 cloud-init Platform, RISC-V support, and Arm64 support.
```

Comment:
@Jehops unless I'm missing something, there's no Arm64 report for 2022q4: <https://github.com/freebsd/freebsd-quarterly/tree/main/2022q4>. 

Amend the sentence here, or leave as is?